ui/StatusQ/src/StatusQ/Popups/Dialog/StatusDialog.qml

```diff
@@ -39,13 +39,20 @@ Dialog {
     */
     property string okButtonText: qsTr("OK")
 
-    readonly property bool bottomSheet: root.contentItem.Window.height
-                                        > root.contentItem.Window.width
-                                        && root.contentItem.Window.width
-                                        <= 1200 // The max width of a phone in portrait mode
+    readonly property bool bottomSheet: d.windowHeight
+                                        > d.windowWidth
+                                        && d.windowWidth
+                                        <= Theme.portraitBreakpoint.width // The max width of a phone in portrait mode
 
     readonly property real desiredY: root.bottomSheet ? root.contentItem.Window.height - root.height : ((root.Overlay.overlay.height - root.height) / 2)
 
+    QtObject {
```

Comment:
@Khushboo-dev-cpp what was the use case for this?
It seems it's breaking some popups on IOS

Reply:
shoot.which ones I had quickly checked and they seemed to all be working okay. Which popup is broken?

If we are explicitly setting height of modal we need to set it as implicit height now. 


I added this excuse after this change for bottom sheet, somehow the popups did not have the correct height detected on android, this made it os so that the popup can take up as much height as they need and don't need to be scrolled so as to make them more usable.

Reply:
Generally speaking the simple popups like the ones in onboarding.

Reply:
I can take a look at this when I am back coming Wednesday, do you want to simply comment out he height part for your demo?

Sorry its causing you inconvenience